pkg/common/auth.go

```diff
@@ -13,6 +13,7 @@ type AuthContext interface {
 	GetAPI() interface{}
 	GetResolvedIdentity() (interface{}, interface{})
 	GetResolvedMetadata() map[interface{}]interface{}
+	ToData() interface{}
```

Comment:
I'm a bit afraid of what the AuthContext is becoming. I'd like to step back and review the design (and specially the name). Probably after we cover the Tier 1 and desired examples.

Reply:
I renamed it to `GetDataForAuthorization()`. Hopefully it is clearer now.